server/src/test/java/org/elasticsearch/action/admin/indices/create/CreateIndexIT.java

```diff
@@ -264,25 +264,6 @@ public void onFailure(Exception e) {
         logger.info("total: {}", expected.getHits().getTotalHits());
     }
 
-    /**
-     * Asserts that the root cause of mapping conflicts is readable.
-     */
-    public void testMappingConflictRootCause() throws Exception {
```

Comment:
while backporting to 6.x, I've reenabled this test (but set `index.version.created` to 5.x)